pkg/vm/fuzz_test.go

```diff
@@ -16,6 +16,7 @@ var fuzzSeedValidScripts = [][]byte{
 	makeProgram(opcode.PUSH2, opcode.NEWARRAY, opcode.DUP, opcode.PUSH0, opcode.PUSH1, opcode.SETITEM, opcode.VALUES),
 	append([]byte{byte(opcode.PUSHDATA1), 10}, randomBytes(10)...),
 	append([]byte{byte(opcode.PUSHDATA1), 100}, randomBytes(100)...),
+	[]byte("4\x0394\x0394\x0394\x0394\x0394\x039C\xbfC\xbfJ\xc1\xbf\xd4C"),
```

Comment:
Care to deobfuscate it?

Reply:
Fuzzing output can be rather intricate, do you think it would be beneficial in some way to have these script implemented with opcodes?

Reply:
Added a simplified version.

Reply:
I know it can, but it's in the code and I have no idea what's going on inside of it, is it a valid code at all? We could at least see what was the problematic case (maybe with some comment like "found by fuzzer, does nasty things").